fj.toml

```diff
@@ -1,9 +1,9 @@
-# The default path that models are loaded from. If the `--model` argument is a
+# The default path that models are loaded from. If the `model` argument is a
```

Comment:
One question to consider: should we uppercase any names of CLI arguments used in the documentation (as it is formatted now in `help`)?

<img width="661" alt="Screenshot 2022-11-07 at 23 54 35" src="https://user-images.githubusercontent.com/26244440/200432728-b35c4249-6295-4790-a49d-92a46a6e2532.png">

Reply:
Good point! Yes, I think we should.

(Still going to merge this as-is though. This is not critical and can always be done in a follow-up PR.)